packages/server/src/launcher.ts

```diff
@@ -59,7 +59,9 @@ export class Launcher {
       port = (await portfinder.getPortPromise()).toString()
     }
 
-    this.serverManager = new ServerManager(this.express.listen(port))
+    const server = this.express.listen(port)
+    this.serverManager = new ServerManager(server)
+    await this.api.sockets.setup(server)
```

Comment:
We will have to make sure that the new server manager (http-terminator) also gracefully closes socket connections!

Reply:
https://linear.app/botpress/issue/MES-132/gracefully-shutdown-websockets-connections